src/System.Device.Ports/System.Device.Ports.SerialPort/SerialData.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Device.Ports.SerialPort
+{
+    /// <summary>
+    /// Specifies the type of character that was received on the serial port of the SerialPort object.
+    /// </summary>
+    public enum SerialData
+    {
+        /// <summary>
+        /// A character was received and placed in the input buffer.
+        /// In the Windows implementation, this is EV_RXCHAR.
+        /// </summary>
+        Chars = 0x01,
+
+        /// <summary>
+        /// The end of file character was received and placed in the input buffer.
+        /// In the Windows implementation, this is EV_RXFLAG.
+        /// </summary>
+        Eof = 0x02
+    }
```

Comment:
Something to think about is a nothing of WatchChar we have introduced in nanoFramework: https://github.com/nanoframework/System.IO.Ports/blob/develop/System.IO.Ports/SerialData.cs.

See usage here: https://github.com/nanoframework/System.IO.Ports#case-of-watchchar

Reply:
I see, this is substantially the same principle used in Win32 for watching a specific character.
Why did you rename the `Eof=0x02` to `WatchChar=0x02` in `SerialData` give they have the same meaning?
If I understand well this enum can be left as-is for compat reasons. Then we can add the `WatchChar` property to specify what is the character that will trigger the event.
Did I miss something?